JavaIslandNewbieVillagePub/missions/install-java.yml

```
@@ -17,7 +17,7 @@ challenges:
   - id: install-java-challenge
     type: Question
     star: 1
-    spec: "(?s).*(jdk|java) version.*"
+    spec: "(?s).*(jdk|java).*"
```

```suggestion
    spec: "(?s).*(jdk version|java version|java \d+).*"
```

只有`jdk|java`似乎有点太宽容了。我会给这个正则写一个测试。